app/Admin/Settings/Fields.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace LevelLevel\VoorDeMensen\Admin\Settings;
+
+class Fields {
+
+	public function register_hooks(): void {
+		add_action( 'admin_init', array( $this, 'register_settings' ) );
+		add_action( 'admin_init', array( $this, 'register_fields' ) );
+	}
+
+	public function register_settings(): void {
```

Comment:
Ik zou de logica hoe dit bestand (en `Sections`) is ingedeeld meer op het functionele groeperen, dan op het technische. 

Nu is het gegroepeerd op het technische (secties opgesplitst van settings, omdat het twee verschillende functies gebruikt) maar ik zou het logischer vinden als er een bestaat `Settings\API` was, dat zowel de section en de velden voor de API instellingen zou aanmaken. Zelfde met `display`.

Laat me weten of je die gedachtegang volgt.

Reply:
ik heb de settings zelf nu leading gemaakt, was inderdaad beter